### PR TITLE
Histogram plot

### PR DESCRIPTION
**Pull request by thempel**
This multi-dimension histogram plot function was requested at the last pyemma workshop after I showed something similar in my presentation. It's nice to have but not very quantitative since histograms are scaled in order to be visible even for many dimensions. It produces graphs like this: 
multidim_hist

### CODE REVIEW

**Comment by franknoe** [comment]
Nice! 


Sent from my T-Mobile 4G LTE Device

-------- Original message --------

File: pyemma/plots/plots1d.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    xyzall : (Concatenated list of) input features; np.array containing time series data to be plotted.
+    feature_labels : Labels of histogrammed features, optional, default = None.
+                  Either list of strings or pyemma Featurizer object.
+    ax : matplotlib Axes object, optional, default = None
+        the axes to plot to. When set to None the default Axes object will be used.
+    ylog : If True, plot logarithm of histogram values.
+    n_bins : Number of bins the histogram uses.
+    outfile : If not None, saves plot to this file.
+    ignore_dim_warning : Enable plotting for more than 50 dimensions (on your own risk).
+    **kwargs: Will be parsed to pyplot.fill_between when plotting the histograms.
```
Comment (marscher) [inline pyemma/plots/plots1d.py]: passed to?

**Comment by marscher** [comment]
Very nice figure :+1: Can you add this fancy new feature to the change log as well? Then we're good to merge.

**Comment by marscher** [comment]
Thank you again! Good job!

**Pull request merged**